easybuild/easyconfigs/c/CUDAcompat/CUDAcompat-11.7-515.48.07.eb

```
@@ -0,0 +1,43 @@
+name = 'CUDAcompat'
+version = '11.7'
+nv_version = '515.48.07'
```

```suggestion
nv_version = '515.65.01'
```
This also has an updated version available